ansible/action_plugins/agnosticd_user_info.py

```diff
@@ -53,9 +65,15 @@ def run(self, tmp=None, task_vars=None):
                     )
                 )
             )
-            fh = open(os.path.join(output_dir, 'user-info.yaml'), 'a')
-            fh.write('- ' + json.dumps(self._task.args['msg']) + "\n")
-            fh.close()
+            if msg:
+                fh = open(os.path.join(output_dir, 'user-info.yaml'), 'a')
```

Comment:
What happens to `user-info.yaml` if ansible-playbook command is run several time ? Do we have the same messages stacked ? Or is the file reset between calls of `ansible-playbook` ?

Reply:
The output directory should start fresh for each run of `ansible-playbook`. The initialization also overwrites the file with `---`.

https://github.com/redhat-cop/agnosticd/blob/babylon-callback-data/ansible/setup_runtime.yml#L33-L40

Reply:
i was asking for  when working from laptop, ansible-playbook is run several times.
in that case, the output_dir is not cleaned up

Reply:
The file is still reset, so should be fine